A37-xds-aggregate-and-logical-dns-clusters.md

```diff
@@ -239,6 +239,15 @@ a single priority with no endpoints. An EDS discovery mechanism should do the
 same if its watcher reports that the resource does not exist (whether or not
 it has previously reported a result).
 
+In an aggregate cluster, the `locality_picking_policy` and
```

Comment:
I think this paragraph actually belongs in the "CDS LB Policy Support for Aggregate and Logical DNS Clusters" section above, since it's talking about a responsibility of the CDS policy, not the xds_cluster_resolver policy.

Since that section is before the description of the xds_cluster_resolver LB policy config, you can either say something like "the `locality_picking_policy` and `endpoint_picking_policy` fields in the `xds_cluster_resolver` policy config (see below)", or you can just avoid mentioning the individual field names and instead say something like "the LB policy config passed down to the `xds_cluster_resolver` policy".